src/client/eppo-client.ts

```diff
@@ -244,6 +248,19 @@ export default class EppoClient {
     banditModelConfigurationStore: IConfigurationStore<BanditParameters>,
   ) {
     this.banditModelConfigurationStore = banditModelConfigurationStore;
+
+    this.maybeUpdateConfigRequestor();
+  }
+
+  private maybeUpdateConfigRequestor() {
```

Comment:
Is "maybe" a common convention for naming functions that contain a conditional?

I would have found something like `updateConfigurationRequestorIfExists` to be more descriptive/self-documenting

Reply:
I like it!